tests/api/test_circulation_utils.py

```diff
@@ -46,3 +47,9 @@ def test_circulation_item_exists(testdata):
 def test_circulation_item_not_exist(testdata):
     """Test return False if item does not exist."""
     assert not circulation_item_exists("not-existing-item-pid")
+
+
+def tes_circulation_is_item_in_loanable_state(testdata):
```

Comment:
typo `tes` -> `test` otherwise your test won't run. Anyway, you can remove it if you remove the method.